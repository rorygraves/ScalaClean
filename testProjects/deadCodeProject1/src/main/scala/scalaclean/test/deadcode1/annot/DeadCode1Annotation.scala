package scalaclean.test.deadcode1.annot

object AnnotApp1 extends App {}

case class UnusedClass(value: String)

@deprecated
case class UnusedClassWithAnnotation(value: String)

/** some docs */
object UnusedObjectDoc
