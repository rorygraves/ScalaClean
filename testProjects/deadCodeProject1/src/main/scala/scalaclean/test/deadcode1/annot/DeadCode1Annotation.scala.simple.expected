package scalaclean.test.deadcode1.annot

object AnnotApp1 extends App {}
